packages/experiments/src/a11y/A11yPage.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import {
```

Comment:
The code in this project probably should be in components/a11y, maybe.

There are no components here. So it relies on calling APIs and passing in resolved elements. So you either scope it to body or you use refs to pull elements out.

It would be nice to:

1. have some component abstraction.
2. have examples using component abstraction.
3. abstract data attribute setting with a helper function so that you can have type safety :)